packages/flutter/lib/src/services/platform_views.dart

```diff
@@ -61,6 +61,11 @@ class PlatformViewsRegistry {
 /// The `id` parameter is the platform view's unique identifier.
 typedef PlatformViewCreatedCallback = void Function(int id);
 
+/// Callback for keyboard navigation moving out of a platform view.
+///
+/// The 'reason' parameter represents the direction in which focus moves.
+typedef LoseFocusCallback = void Function(int reason);
```

Comment:
If it's specific to keyboard navigation, it seems like this should have a more specific name.